src/preferences/dialog/dlgprefinterfacedlg.ui

```diff
@@ -29,7 +29,7 @@
          <font/>
         </property>
         <property name="text">
-         <string>Skin</string>
+         <string>S&amp;kin</string>
```

Comment:
String changed by accident?

Reply:
QtDesigner did this automatically. I can rebase and remove the noise from the diff.